documentation/source/users/arkane/input.rst

```diff
@@ -863,7 +863,22 @@ Use a ``kinetics()`` function to make Arkane execute the high-pressure limit kin
 parameters computation for a reaction. The ``'label'`` string must correspond to that of
 a defined ``reaction()`` function.
 
-You have three options for specifying the temperature to which a modified Arrhenius
+By default, Arkane uses a modified three-parameter Arrhenius equation to fit kinetics data:
+
+  .. math :: k(T) = A \exp \left( -\frac{E_a}{RT} \right) \left( \frac{T}{T_0} \right)^n
+
+Alternatively, you have the freedom to use the classic two-parameter Arrhenius equation:
```

Comment:
suggestion:
"Alternatively, the user may request to output the rate in the classical two-parameter Arrhenius format:"

Reply:
ok